modules/db_instance/variables.tf

```diff
@@ -39,6 +39,10 @@ variable "port" {
   description = "The port on which the DB accepts connections"
 }
 
+variable "final_snapshot_identifier" {
+  description = "The name of your final DB snapshot when this DB instance is deleted."
+}
```

Comment:
There should be `default = ""`, because this is an optional argument.

Reply:
Since skip_final_snapshot defaults to false (https://github.com/terraform-aws-modules/terraform-aws-rds/blob/master/modules/db_instance/variables.tf#L108) And the AWS api wont allow you to delete an RDS instance without a final_snapshot_identifier set unless skip_final_snapshot = true, I'm not sure what is the best setting here.  It seems to be that either skip_final_snapshot should be true by default, which I would consider a bad design decision as it could have unwanted effects.  So my thought was to force the identification of final_snapshot_identifier instead.

Reply:
This resource module does not have to be smarter than Terraform & AWS API, so keeping it as `default = ""` is a good solution. Please fix and we merge this PR.

Reply:
👍